src/test/java/org/takes/facets/fork/FkContentTypeTest.java

```diff
@@ -156,4 +167,17 @@ public void equalsAndHashCodeEqualTest() throws Exception {
             .suppress(Warning.TRANSIENT_FIELDS)
             .verify();
     }
+
+    /**
+     * Create a Take instance with empty response.
+     * @return Take
+     */
+    private static Take createTakeWithEmptyResponse() {
```

Comment:
@dalifreire read this: http://www.yegor256.com/images/books/elegant-objects/seven-pages.pdf